Core/Sources/Configuration/RoutingProfile.swift

```diff
@@ -1,3 +1,5 @@
+// swiftlint:disable trailing_whitespace
```

Comment:
Do you need this, given you've disabled this also in `.swiftlint.yml`?

Same goes for other locations where you're explicitly overriding this rule via commentary.